app/views/items/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<p> <%= flash[:error] %> </p>
+
+<%= render 'item_form' %>
+
+<%= render 'category_form' %>
+
+<table class='table table-bordered'>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Price</th>
+      <th>Quantity</th>
+      <th>Expires</th>
+    </tr>
+  </thead>
+  <tbody>
+  <% Item.valid_items.each do |item| %>
```

Comment:
The `Item.valid_items` should move back to the controller here, and be passed through a `@items` instead, so there's even less logic in the view.